src/components/Error/style.scss

```diff
@@ -40,53 +39,50 @@
   display: inline-block;
   font-weight: 600;
   line-height: 1.5;
-  padding-left: 24*@unit;
+  padding-left: 24*$unit;
 }
 
 
 .title-icon {
   position: absolute;
   height: 100%;
-  width: 16*@unit;
-  margin-right: 8*@unit;
+  width: 16*$unit;
+  margin-right: 8*$unit;
   background-position: center;
   background-repeat: no-repeat;
   background-size: contain;
 }
 
 .title-icon-error {
-  &:extend(.title-icon);
   background-image: url('assets/error.svg');
 }
 
 .title-icon-warning {
-  &:extend(.title-icon);
   background-image: url('assets/warning.svg');
 }
 
 .roundedTriangle {
   position: absolute;
-  bottom: -6*@unit;
-  left: ~"calc(50% - 8*@{unit})";
-  width: 8*@unit;
-  height: 8*@unit;
-  border-top: 8*@unit solid @color-transparent;
-  border-right: 8*@unit solid @color-transparent;
-  border-bottom: 8*@unit solid @color-transparent;
-  border-left: 8*@unit solid @color-transparent;
-  border-bottom-left-radius: 3*@unit;
+  bottom: -6*$unit;
+  left: calc(100% - #{8*$unit});
```

Comment:
This should be `calc(50% - #{8*$unit});` otherwise the triangle at the bottom of the error is out of position
![Screenshot 2020-07-10 at 11 02 58](https://user-images.githubusercontent.com/2497081/87142772-ec197700-c29c-11ea-94c1-fadcdf3d918c.png)

Reply:
Good catch, thanks!